src/SharedLibrary/GeneratedCode/SharedLibrary_Common.h

```diff
@@ -50,6 +50,36 @@ FEATURIZER_LIBRARY_API bool DestroyErrorInfoString(/*in*/ char const *input_ptr,
 FEATURIZER_LIBRARY_API bool DestroyErrorInfo(/*in*/ ErrorInfoHandle *pHandle);
 FEATURIZER_LIBRARY_API bool DestroyTransformerSaveData(/*in*/ unsigned char const *pBuffer, /*in*/ std::size_t cBufferSize, /*out*/ ErrorInfoHandle **ppErrorInfo);
 
+FEATURIZER_LIBRARY_API_PACK_PREFIX;
+
+struct DateTimeParameter {
+    // ----------------------------------------------------------------------
+    // |  Public Types
+    enum DateTimeTypeValue {
+        DateTimeInt64,      // Posix time

```

Comment:
Can we add explicit numbers to these starting with 1 so it matches the other enums we have?